include/nuttx/power/bq27426.h

```diff
@@ -0,0 +1,227 @@
+#define BQ27426_CONTROL_CHEM_B          0x0031
+#define BQ27426_CONTROL_CHEM_C          0x0032
+#define BQ27426_CONTROL_RESET           0x0041
+#define BQ27426_CONTROL_SOFT_RESET      0x0042
+
+/* Control Status Word - Bit Definitions */
+
+/* Bit positions for the 16-bit data of CONTROL_STATUS.
+ * CONTROL_STATUS instructs the fuel gauge to return status
+ * information to Control() addresses 0x00 and 0x01.
+ * The read-only status word contains status  bits that are
+ * set or cleared either automatically as conditions warrant or
+ * through using specified subcommands.
+ */
+
+#define BQ27426_STATUS_SHUTDOWNEN    (1 << 15)
+#define BQ27426_STATUS_WDRESET       (1 << 14)
+#define BQ27426_STATUS_SS            (1 << 13)
+#define BQ27426_STATUS_CALMODE       (1 << 12)
+#define BQ27426_STATUS_CCA           (1 << 11)
+#define BQ27426_STATUS_BCA           (1 << 10)
+#define BQ27426_STATUS_QMAX_UP       (1 << 9)
+#define BQ27426_STATUS_RES_UP        (1 << 8)
+#define BQ27426_STATUS_INITCOMP      (1 << 7)
+#define BQ27426_STATUS_SLEEP         (1 << 4)
+#define BQ27426_STATUS_LDMD          (1 << 3)
+#define BQ27426_STATUS_RUP_DIS       (1 << 2)
+#define BQ27426_STATUS_VOK           (1 << 1)
+#define BQ27426_STATUS_CHEMCHANGE    (1 << 0)
+
+/* Flag Command - Bit Definitions */
+
+/* Bit positions for the 16-bit data of Flags()
+ * This read-word function returns the contents of the fuel gauging status
+ * register, depicting the current operating status.
+ */
+
+#define BQ27426_FLAG_OT            (1 << 15)
+#define BQ27426_FLAG_UT            (1 << 14)
+#define BQ27426_FLAG_FC            (1 << 9)
+#define BQ27426_FLAG_CHG           (1 << 8)
+#define BQ27426_FLAG_OCVTAKEN      (1 << 7)
+#define BQ27426_FLAG_DODCORRECT    (1 << 6)
+#define BQ27426_FLAG_ITPOR         (1 << 5)
+#define BQ27426_FLAG_CFGUPMODE     (1 << 4)
+#define BQ27426_FLAG_BAT_DET       (1 << 3)
+#define BQ27426_FLAG_SOC1          (1 << 2)
+#define BQ27426_FLAG_SOCF          (1 << 1)
+#define BQ27426_FLAG_DSG           (1 << 0)
+
+/* Extended Data Commands */
+
+/* Extended data commands offer additional functionality beyond
+ * the standard set of commands.
+ * They are used in the same manner; however, unlike standard
+ * scommands, extended commands are not limited to 2-byte words.
+ */
+
+#define BQ27426_EXTENDED_OPCONFIG   0x3A /* OpConfig()*/
+#define BQ27426_EXTENDED_CAPACITY   0x3C /* DesignCapacity()*/
+#define BQ27426_EXTENDED_DATACLASS  0x3E /* DataClass()*/
+#define BQ27426_EXTENDED_DATABLOCK  0x3F /* DataBlock()*/
+#define BQ27426_EXTENDED_BLOCKDATA  0x40 /* BlockData()*/
+#define BQ27426_EXTENDED_CHECKSUM   0x60 /* BlockDataCheckSum()*/
+#define BQ27426_EXTENDED_CONTROL    0x61 /* BlockDataControl()*/
+
+/* Configuration Class, Subclass ID's */
+
+/*  To access a subclass of the extended data, set the DataClass()
+ *  function with one of these values.
+ */
+
+/* Configuration Classes */
+
+#define BQ27426_ID_SAFETY            2  /* Safety */
```

Comment:
Please, try to align those value to start all in the same column, this block and all others